bigtable/examples/run_examples_emulator.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+readonly BINDIR=$(dirname $0)
+source ${BINDIR}/run_examples_utils.sh
+source ${BINDIR}/../tools/run_emulator_utils.sh
```

Comment:
Quote the full file name path (https://google.github.io/styleguide/shell.xml#Quoting).

Reply:
Done